cluster-up/cluster/kind/common.sh

```diff
@@ -3,7 +3,7 @@
 set -e
 
 NODE_CMD="docker exec -it -d "
-export KIND_MANIFESTS_DIR="${KUBEVIRTCI_PATH}/cluster/kind/manifests"
+export KIND_MANIFESTS_DIR="cluster-up/cluster/kind/manifests"
```

Comment:
Why this changed? Relative paths are risky.

Reply:
When using `yq` inside a container we mount kubevirtci path as the working dir inside the container.
Now wen we pass a file with a full path it fails because `yq` cant find that file, its out of the container scope.
So I changed `KIND_MANIFESTS_DIR` to relative path and checked that it dosent break anything (all check provision passed as well) then `yq` worked